test/cqlpy/test_tablets.py

```diff
@@ -323,3 +323,33 @@ def change_dc_rf(new_rf):
             change_dc_rf(10)  # increase RF by 2+ should fail
         with pytest.raises(InvalidRequest):
             change_dc_rf(0)  # decrease RF by 2+ should fail
+
+
+def test_tablet_hints(cql, skip_without_tablets):
+    def describe_table(cql, table):
+        return cql.execute(f"DESC TABLE {table}").one().create_statement
+
+    ksdef = "WITH REPLICATION = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1} AND TABLETS = {'enabled': false};"
```

Comment:
Please add a comment in front of this test, or in front of this line, explaining what it's trying to. It left me confused - this lines says that tablets are NOT enabled. So what do "tablet hints" do here, and why?

Reply:
sure

Reply:
Hmm, that's actually a bug, I think.
tablet_hints should be rejected when tablets are disabled for the keyspace.
To enable them, one has to enable tablets for the keyspace.

Reply:
fixed in v3, add a separate negative test